app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -88,3 +88,5 @@ CONFIG_CLOCK_CONTROL=y
 CONFIG_DEBUG_COREDUMP=y
 CONFIG_DEBUG_COREDUMP_BACKEND_INTEL_ADSP_MEM_WINDOW=y
 CONFIG_DEBUG_COREDUMP_MEMORY_DUMP_MIN=y
+
+CONFIG_KCPS_DYNAMIC_CLOCK_CONTROL=y
```

Comment:
I would suggest N this here for now, until peak mcps rootcaused, it is 3 times larger than avg.

Reply:
Yes, we can drop this commit before merge but for now its good to turn it on and test in CI

Reply:
@btian1 Let's turn this on, so we can test this. The peak is not necessarily a problem if the SRAM buffering we have can cover it. If not, then we need to anyways address this, either fix the rootcause for peak and/or increase the budgets in manifest.

Reply:
ok